### PR TITLE
cleanup: jobs housekeeping p3

### PR DESCRIPTION
**Pull request by Ressetkk**
/kind cleanup
/area ci

Cleanup of jobs definition - moving to org/repo structure and update CODEOWNERS file.

#7386

### CODE REVIEW

**Review by Cortey — APPROVED** [review]

File: prow/jobs/kyma-project/test-infra/images.yaml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-# Code generated by rendertemplates. DO NOT EDIT.
```
Comment (ralikio) [inline prow/jobs/kyma-project/test-infra/images.yaml:1]: Is the change intentional?

**Review by ralikio — APPROVED** [review]

File: prow/jobs/kyma-project/test-infra/images.yaml (hunk 1 above)
Comment (Ressetkk) [reply prow/jobs/kyma-project/test-infra/images.yaml:1]: Yes.

File: CODEOWNERS [hunk 2]
```diff
@@ -22,73 +21,67 @@
...
+/prow/jobs/kyma-project/kyma/periodics.yaml @kyma-project/prow @neighbors-dev-bot
+/prow/jobs/kyma-project/kyma/skr-aws-upgrade-integration-dev.yaml @kyma-project/gopher @neighbors-dev-bot
+/prow/jobs/kyma-project/kyma/skr-integration.yaml @kyma-project/gopher @neighbors-dev-bot
 
-/prow/jobs/kyma-dashboard @kyma-project/hasselhoff @neighbors-dev-bot
-/prow/jobs/kyma-environment-broker @kyma-project/gopher @neighbors-dev-bot
+/prow/jobs/kyma-project/kyma-dashboard @kyma-project/hasselhoff @neighbors-dev-bot
+/prow/jobs/kyma-project/kyma-environment-broker @kyma-project/gopher @neighbors-dev-bot
 /prow/jobs/kyma-project/compass-manager @kyma-project/framefrog @neighbors-dev-bot
 /prow/jobs/kyma-project/control-plane @kyma-project/framefrog @neighbors-dev-bot
```
[the remark this answers is not in the document]
Comment (Ressetkk) [reply CODEOWNERS:68]: It is. I wonder who added that.

[the remark this answers is not in the document]
Comment (Ressetkk) [reply CODEOWNERS:68]: Eh. I'll fix that eventually

**Review by mvshao — DISMISSED** [review]

**Review by k15r — APPROVED** [review]

**Pull request merged**